core/trino-main/src/main/java/io/trino/sql/planner/ConnectorExpressionTranslator.java

```diff
@@ -883,19 +882,5 @@ private Type typeOf(Expression node)
         {
             return types.get(NodeRef.of(node));
         }
-
-        private Object evaluateConstant(Expression node)
```

Comment:
Would be nice to have a static code analysis to catch cases like that.
unused (private) methods looks like something easy to find

Reply:
Can you please point me to where we define global static code analysis?

Reply:
spotbugs, error-prone, checkstyle, ... look in top-level pom.xml and in airbase pom too

Reply:
Thanks, I think it can be done on a separate PR

Reply:
sure, looking forward!